riscv/src/compiler.rs

```diff
@@ -967,6 +1031,10 @@ fn process_instruction(instr: &str, args: &[Argument]) -> Vec<String> {
             let (rd, r1, r2) = rrr(args);
             only_if_no_write_to_zero(format!("{rd} <=Z= divu({r1}, {r2});"), rd)
         }
+        "remu" => {
+            let (rd, r1, r2) = rrr(args);
+            only_if_no_write_to_zero(format!("{rd} <=Z= remu({r1}, {r2});"), rd)
```

Comment:
I think Z is redundant here

Reply:
But is more symmetric, given just above Z is used.